README.md

```diff
@@ -5,3 +5,4 @@ Patterns and sugar for ROS2 parameters
 
 * parameter_set - C++ library for standardizing declaring, validating, getting, and handling of dynamic parameters
 * parameter_builder - Python library for loading parameters in launch files
+* third_party/absl_status - Refactored status library from absl
```

Comment:
What part of the file got changed .? can you include a patch file or something just in case so we don't forget what was changed

Reply:
I had to modify it fairly heavily to remove dependencies on other parts of absl.  Long term I want to use a vendor or Debian package for absl and not this.  This is only here as a hack to support other work until I can figure out how to depend on absl in a better way.

Reply:
I converted to using a PPA for absl for now so this refactored version of statusor is gone.